covid19_sfbayarea/news/contra_costa.py

```diff
@@ -101,6 +101,14 @@ def parse_page(self, html: str, url: str) -> List[NewsItem]:
                 if item:
                     news.append(item)
 
+        # TODO: Figure this one out! It's possible this could be as simple as
+        # using an actual browser (e.g. Selenium) instead of requests.
+        if len(news) == 0:
```

Comment:
```suggestion
        if not news:
```
https://www.python.org/dev/peps/pep-0008/#programming-recommendations
> For sequences, (strings, lists, tuples), use the fact that empty sequences are false